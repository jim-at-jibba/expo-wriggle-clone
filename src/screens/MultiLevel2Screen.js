import React from 'react';
import { ScrollView, Text, View } from 'react-native';
import { utilStyles } from '../api/constants';

// grab component
import NavigationBack from '../components/NavigationBack';

const MultiLevel2Screen = () => (
  <View style={utilStyles.container}>
    <ScrollView contentContainerStyle={utilStyles.contentContainer}>
      <Text style={utilStyles.text}>Multi Screens :: Level 2 Content Area</Text>
    </ScrollView>
  </View>
);

MultiLevel2Screen.navigationOptions = ({ navigation }) => ({
  headerLeft: <NavigationBack navigation={navigation} />,
  headerRight: <View style={{ flex: 1 }} />,
  headerTitleStyle: {
    flex: 1,
    textAlign: 'center'
  },
  title: 'Level 2'
});

export default MultiLevel2Screen;
