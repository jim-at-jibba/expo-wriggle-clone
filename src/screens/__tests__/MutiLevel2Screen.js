import React from 'react';
import renderer from 'react-test-renderer';

import MultiLevel2Screen from '../MultiLevel2Screen';

const navigation = {
  goBack: jest.fn(),
  navigate: jest.fn()
};

const tree = renderer
  .create(<MultiLevel2Screen navigation={navigation} />)
  .toJSON();

describe('<MultiLevel2Screen />', () => {
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
