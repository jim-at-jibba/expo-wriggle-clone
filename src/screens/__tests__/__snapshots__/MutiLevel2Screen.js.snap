// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultiLevel2Screen /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
/>
`;
